blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -558,13 +519,13 @@ export const actions = {
                 // We're only interested in this event if we're already managing branch state
                 // associated with this multi-branch job.
                 if (!multibranchPipelines[pipelineName]) {
-                    return;
+                    return Promise.resolve(null);
```

Comment:
yes and isomorphic-fetch too, its on my todo list :). Was reading about polyfill last night after I saw another commment about it